CHANGELOG.md

```diff
@@ -74,6 +74,9 @@ Keep it human-readable, your future self will thank you!
 ### Changed
 
 - Updated configuration examples in documentation and corrected links - [#46](https://github.com/ecmwf/anemoi-training/pull/46)
+- Refactored callbacks. [#60](https://github.com/ecmwf/anemoi-training/pulls/60)
+- Remove credential prompt from mlflow login, replace with seed refresh token via web - [#78](https://github.com/ecmwf/anemoi-training/pull/78)
+- Refactored callbacks. [#60](https://github.com/ecmwf/anemoi-training/pulls/60)
```

Comment:
This is likely to be an issue, can you remove the incorrect updates

Reply:
should be good now with entry for this PR